hack/netpol/pkg/main/main.go

```diff
@@ -274,7 +274,8 @@ func executeTests(k8s *Kubernetes, testList []*TestCase) []*TestCase {
 	return modifiedTestList
 }
 
-/**
+/*
+*
```

Comment:
out of curiosity, does the second "*" mean anything?

Reply:
it doesn't mean anything. `gofmt` now does some comment reformatting, and this file was updated automatically. I am ignoring this change since this whole folder (`hack/netpol`) should be removed pretty soon.